test/Performance/run-perf_tests.py

```diff
@@ -0,0 +1,792 @@
+    On Windows, this is done by modifying connect.php file.
+    On Linux and Mac, odbcinst.ini file needs to be modified.
+    @TO-DO: Currently modifying odbcinst.ini requires root permissions.
+    Copy the MSODBCSQL to a location which does not require sudo. 
+    """
+    print( "Enabling Pooling...")
+    if os.name == 'nt':
+        with fileinput.FileInput( connect_file, inplace=True, backup='.bak') as file:
+            for line in file:
+                print( line.replace( "$pooling=false;", "$pooling=true;" ), end='')
+    else:
+        # Get the location of odbcinst.ini
+        odbcinst = os.popen( "odbcinst -j" ).read().splitlines()[1].split()[1]
+        odbcinst_bak = odbcinst + ".bak"
+ 
+        # Create a copy of odbcinst.ini
+        copyfile( odbcinst, odbcinst_bak )
+ 
+        # Lines to enable Connection pooling
+        lines_to_append="CPTimeout=5\n[ODBC]\nPooling=Yes\n"
+ 
+        with open( odbcinst, "a" ) as f:
+            f.write( lines_to_append )
+ 
+def disable_pooling():
+    """
+    This module disables Connection Pooling. 
+    On Windows, this is done by modifying connect.php file.
+    On Linux and Mac, odbcinst.ini file needs to be modified.
+    @TO-DO: Currently modifying odbcinst.ini requires root permissions.
+    Copy the MSODBCSQL to a location which does not require sudo. 
+    """
+    print("Disabling Pooling...")
+    if os.name == 'nt':
+        os.remove( connect_file )
+        copyfile( connect_file_bak, connect_file )
+    else:
+        # Get the location of odbcinst.ini
+        odbcinst = os.popen( "odbcinst -j" ).read().splitlines()[1].split()[1]
+        odbcinst_bak = odbcinst + ".bak"
+        os.remove( odbcinst )
+        copyfile( odbcinst_bak, odbcinst )
+        os.remove( odbcinst_bak )
+ 
+def run_tests( iterations, iterations_large ):
+    """
+    This module runs the tests using PHPBench
+    Args:
+        iterations (int): Number of iterations the tests in "regular" folder are run for
+        iterations_large (int): Number of iterations the tests in "large" folder are run for
+    Returns:
+        N/A
+    """
+    print("Running the tests...")
+    call( get_run_command( sqlsrv_regular_path, iterations, "sqlsrv-regular.xml" ), shell=True )
+    call( get_run_command( sqlsrv_large_path, iterations_large, "sqlsrv-large.xml" ), shell=True )
+ 
+    call( get_run_command( pdo_regular_path, iterations, "pdo_sqlsrv-regular.xml" ), shell=True )
+    call( get_run_command( pdo_large_path, iterations_large, "pdo_sqlsrv-large.xml" ), shell=True )
+ 
+def parse_results( dump_file ):
+    """
+    This module parses the .xml files generated by PHPBench
+    @TO-DO: Currently only limited detailes are parsed, such as duration and peak memory.
+    PHPBench reports a lot more information that can be helpful.
+    Consider looking at the xml files.
+    Args:
+        dump_file (str): The name of the XML file to be parsed.
+    Returns:
+        N/A

```

Comment:
returns array of parsed results, each entry has information like duration and memory